src/project.ts

```diff
@@ -96,6 +97,66 @@ function hideEditor (program: Program): void {
 	}
 }
 
+//Replace KA's vote button with one that updates after you vote and allows undoing votes
+function replaceVoteButton (program: Program): void {
+	querySelectorPromise(".discussion-meta-controls")
+	.then(wrap => {
+		if (!wrap || !(wrap.firstChild as HTMLElement).innerText.includes("Vote")) {
+			return;
+		}
+
+		wrap = wrap.firstChild!;
+
+		const unvotedClass =    "link_1uvuyao-o_O-computing_77ub1h";
+		const votedClass =      "link_1uvuyao-o_O-computing_1w8n1i8";
+
+		const voteURL = "https://www.khanacademy.org/api/internal/discussions/voteentity";
+
+		let voted = (wrap as HTMLElement).innerText.includes("Voted Up");
+
+		const orgVotes = program.sumVotesIncremented - (voted ? 1 : 0);
+
+		const newWrap = document.createElement("span");
+		const voteButton = document.createElement("a");
+		const voteText = document.createElement("span");
+		voteButton.appendChild(voteText);
+		newWrap.appendChild(voteButton);
+
+		function updateVoteDisplay() {
```

Comment:
It might seem awkward, but our style guide mandates that there be a space between the function identifier and the parentheses.  This small style difference is causing Travis to reject this PR.